google/cloud/forseti/services/model/model.proto

```diff
@@ -45,10 +45,7 @@ message DeleteModelRequest {
 }
 
 message DeleteModelReply {
-  enum Status {
-    SUCCESS = 0;
-  }
-  Status status = 1;
+  string result = 1;
```

Comment:
Instead of deleting the status field, can you please keep it and add a `FAIL = 1`.
Can you also rename the `result` field, call it `message = 2`, and use it to send details of the operation?  model name of what has been deleted, or what exception has occurred.